src/main/java/by/itechart/api/dto/UserDTO.java

```diff
@@ -9,6 +9,47 @@
 @EqualsAndHashCode(callSuper = true)
 @NoArgsConstructor
 @Data
+@ApiModel(value = "User model")
 public class UserDTO extends UpdateUserDTO {
+    @ApiModelProperty(
+            value = "User ID",
+            name = "id",
+            dataType = "Long",
+            example = "19485",
+            readOnly = true)
     Long id;
+
+    @ApiModelProperty(
+            value = "User's first name",
+            name = "firstName",
+            dataType = "String",
+            example = "Adam")
+    String firstName;
+
+    @ApiModelProperty(
+            value = "User's last name",
+            name = "lastName",
+            dataType = "String",
+            example = "Smith")
+    String lastName;
+
+    @ApiModelProperty(
+            value = "User's email",
+            name = "email",
+            dataType = "String",
+            example = "adam.smith@gmail.com",
+    required = true)
+    String email;
+
+    @ApiModelProperty(
+            notes = "User's password should not be exposed",
+            hidden = true)
```

Comment:
Also, why here password is hidden? The user can change his password. What do you think?

Reply:
Here is [why](https://github.com/Dauren-Delmukhambetov/legendary-succotash/pull/21#discussion_r602390545)